packages/components/table/components/messages/xm-table-warning-message.component.scss

```
@@ -0,0 +1,84 @@
+    animation: table-warning-message-slide-out 0.25s ease-in forwards;
+  }
+
+  .close-icon {
+    margin-left: auto;
+    cursor: pointer;
+    width: 28px;
+    height: 28px;
+    padding: 0;
+    flex-shrink: 0;
+
+    ::ng-deep .mat-mdc-button-touch-target {
```

Stylelint (17.14.0)

[error] Unknown pseudo-element selector "::ng-deep" (selector-pseudo-element-no-unknown)

(selector-pseudo-element-no-unknown)